programs/uxd/tests/integration_tests/api/program_uxd/instructions/process_collect_profits_of_mercurial_vault_depository.rs

```diff
@@ -190,19 +191,23 @@ pub async fn process_collect_profits_of_mercurial_vault_depository(
         )
         .unwrap();
 
-    // Check result
+    // vault_lp_mint.supply must have decreased by the withdrawn profit amount
```

Comment:
This is an attempt to improve readability by:
- separating chunks of codes
- quick description of what we are looking for checking

Reply:
Based on @acamill feedback on last PR about huge chunks of code being unreadable (which I kinda agree with)